monitoring/ceph-mixin/prometheus_alerts.yml

```diff
@@ -652,3 +652,45 @@ groups:
           oid: "1.3.6.1.4.1.50495.1.2.1.1.2"
           severity: "critical"
           type: "ceph_default"
+  - name: "rbdmirror"
+    rules:
+      - alert: "CephRBDMirrorImagesPerDaemonHigh"
+        annotations:
+          description: "Number of image replication per daemon is not suppossed to go beyond the provided threshold"
+          summary: "Number of image replication is now above 100"
```

Comment:
"Number of image replication is now above %(CephRBDMirrorImagesPerDaemonThreshold)"

Reply:
This is the generated yaml, @jmolmo . It will always show the value.